src/components/Gradebook/index.jsx

```diff
@@ -831,16 +842,14 @@ export default class Gradebook extends React.Component {
                 onChange={this.handleMaxAssigGradeChange}
               />
               <span className="input-percent-label">%</span>
-              <div className="d-flex align-items-center">
```

Comment:
What does getting rid of this `<div>` do?

Reply:
Ah, you just added a screenshot that answers this question :)

Reply:
The div was an attempt to get the button into a better-aligned position, but it also complicated efforts to fix this bug. I figure it's also good to make the assignment grade filter and the course grade filter markup consistent with one another, also.